connaisseur/tests/integration/LOCAL.md

```diff
@@ -0,0 +1,66 @@
+# Local integration test setup
+
+## Minikube
+
+To load the required docker images into the minikube docker daemon run
+
+```shell
+eval $(minikube docker-env)
+make docker
+```
+
+which will change the environment variables that specify the docker daemon to point to minikube instead of your system docker daemon.
+
+### Preparation for minikube with virtualbox driver
+
+Assuming you have a running minikube cluster using the virtualbox driver set as current kubernetes context you just need to have the alerting interface running.
+
+Run the docker container that serves as alerting endpoint and retrieve the IP address that it has on the `bridge` network:
+
+```shell
+docker run -d --name alerting-endpoint -p 56243:56243 docker.io/securesystemsengineering/alerting-endpoint:latest
+export ALERTING_ENDPOINT_IP=$(docker container inspect alerting-endpoint | jq -r .[].NetworkSettings.Networks.bridge.IPAddress)
+```
+
+### Preparation for minikube with docker driver
+
+Assuming you have a running minikube cluster using the docker driver set as current kubernetes context you need to have the alerting interface running and attach the container to the network that is used by minikube to facilitate communication. By default, this network is named `minikube`, so change the minikube network name to your specific name if you use a customized one:
+
+```shell
+MINIKUBE_NETWORK=minikube
+docker run -d --name alerting-endpoint -p 56243:56243 docker.io/securesystemsengineering/alerting-endpoint:latest
+docker network connect ${MINIKUBE_NETWORK} alerting-endpoint
+export ALERTING_ENDPOINT_IP=$(docker container inspect alerting-endpoint | jq -r --arg minikube_network ${MINIKUBE_NETWORK} '.[].NetworkSettings.Networks[$minikube_network].IPAddress')
+
+```
+
+### Actual test
+
+From the git repository folder run the `connaisseur/tests/integration/integration-test.sh` script.
+
+To cleanup the mocked alerting interface container don't forget running 
+
+```shell
+docker stop alerting-endpoint
+docker rm alerting-endpoint
+```
+
+## Kind
+
+For kind we assume you have a running kind cluster set as current kubernetes context and that you have loaded the docker images from the latest connaisseur image version as well as the hook image onto the `kind` nodes.
```

Comment:
Maybe add the `kind load docker-image` to this paragraph to tell people how to load the images